cms/static/js/spec/views/course_video_settings_spec.js

```diff
@@ -599,14 +638,17 @@ define(
             });
 
             it('shows error message on saving organization credentials if server sends error', function() {
-                var requests = AjaxHelpers.requests(this);
+                var requests = AjaxHelpers.requests(this),
+                    apiKey = 'test-api-key',
```

Comment:
This method and above method can be merged. We can write a single method to verifies for both cielo24 and 3play that credentials are saved and correct preferences view is rendered upon successful credentials update.